sdk/src/wallet/core/mod.rs

```diff
@@ -213,15 +456,15 @@ impl<S: SecretManage> WalletInner<S> {
     }
 
     #[cfg(feature = "events")]
```

Comment:
Im thinking maybe we want to emit with alias? or add to the alias questions

Reply:
Open an issue, but that doesn't strike me as a very useful feature. It was needed for users to know which account the event was from but now there is only one thing that could emit it.

Reply:
yea but are we not making this with the idea that users create themselves multiple wallets? odds are, the same method isused to receive events for all wallets.

Reply:
https://github.com/iotaledger/iota-sdk/issues/1544